supernovacontroller/utils/status_codes.py

```diff
@@ -0,0 +1,14 @@
+class CodeTranslator:
+    i2c_codes = {
+        0 : "success",
+        51 : "NACK_ERROR"
+    }
+    library = {
+        "i2c" : i2c_codes
+    }
+    
+    def get_message(protocol, code):
```

Comment:
Maybe add `@staticmethod` to `get_message` to make it more explicit?

Reply:
I agree with this comment